packages/go_router_builder/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 1.0.1
+
+- Changed the argument name of the auto-generated `go` method from `buildContext` to `context`.
```

Comment:
```suggestion
- Changes the argument name of the auto-generated `go` method from `buildContext` to `context`.
```

Reply:
Usually put changelog entries in the past tense, right?

Reply:
The last time I was asked to put present tense.

Reply:
Is this a @Hixie opinion thing?

Reply:
Relevant: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style

TL;DR: "...use present tense indicative for verbs..."